TUTORIAL.md

```diff
@@ -0,0 +1,383 @@
+- Node.js installed. We recommend using the latest LTS version.
+
+## Initialize an app with npm
+
+1. Create a new directory for your app.
+2. Open a console window and navigate to the directory.
+3. Run `npm init` and enter in the requested details about your application. If you are unsure about any prompts, press `Enter` to use the default value. You can modify these details at any point in the `package.json` file that `npm init` creates.
+
+## Create the app code
+
+1. Create a `src` directory for your app code. When we build the app, webpack will automatically produce distribution files in a directory named `dist`.
+2. Create the files `index.html` and `index.js` in the `src` directory added in step 1.
+3. Add the following code to `src/index.html`
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <p>Hello World!</p>
+  </body>
+</html>
+```
+This will serve as a boilerplate HTML page to get us started.
+4. Add the following code to `src/index.js`
+```
+console.log('Hello World!');
+```
+This will serve as our boilerplate JavaScript code.
+
+## Install and configure webpack
+1. Install webpack by running `npm install --save-dev webpack`. The files `package-lock.json` and `package.json` should appear in your main directory. You should also see a new folder in your main directory named `node_modules`.
+2. Create the file `webpack.config.js` in your main directory. 
+3. Define our webpack [configuration](https://webpack.js.org/concepts/configuration/) object by adding the following code to `webpack.config.js`
+```
+const path = require('path');
+
+const webpack = require('webpack');
+
+module.exports = {
+    context: __dirname,
+    entry: {
+        app: './src/index.js'
+    },
+    output: {
+        filename: 'app.js',
+        path: path.resolve(__dirname, 'dist'),
+    }
+};
+```
+In this code, `context` specifies the base path for your files. `entry` is used to specify bundles and `src/index.js` is our entry point. Webpack will output the bundel `app.js` to the folder `dist`, that webpack will create at runtime.
+4. Webpack loads everything like a module. [loaders](https://webpack.js.org/concepts/#loaders) are used to load CSS and other asset files. Install the [style-loader](https://webpack.js.org/loaders/style-loader/#src/components/Sidebar/Sidebar.jsx), [css-loader](https://webpack.js.org/loaders/css-loader/), and [url-loader](https://webpack.js.org/loaders/url-loader/) using `npm install --save-dev style-loader css-loader url-loader`. Feel free to install any other loaders you may need in the future. Loaders can be installed at any point during this process.
+5. Update `webpack.config.js` by adding two `module.rules`. The first rule should support CSS files and the second rule should support other static files. For each rule, define `test` for the types of files to load and `use` to specify the list of loaders. `webpack.config.js` should look something like
+```
```

Comment:
Not sure what the fix is but I noticed the numbering loses continuity here: 

![image](https://user-images.githubusercontent.com/31491650/136324223-01477f6a-17dd-4a3e-81f2-8cc9f46b3ae1.png)